releasenotes/notes/ddsketch-api-a84ffc0875bbacd6.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - "``DDSketch`` attributes ``mapping``, ``store``, ``negative_store``, ``zero_count``, ``relative_accuracy``, ``min`` and ``max`` have been made internal."
+  - "``DDSketch.copy`` has been made internal."
+  - "``DDSketch.mergeable`` has been made internal."
```

Comment:
why do this as a string and not a multiline string?

Reply:
Multiline doesn't seem to pick up on the bullet points